db/migrate/20201027221804_install_audited.rb

```diff
@@ -0,0 +1,30 @@
+class InstallAudited < ActiveRecord::Migration[6.0]
```

Comment:
Oh - can you update the date of this migration (just the file name)? Because right now it's before the most recent migration since it was done awhile ago. It was auto-generated by the audit gem so you can remove the migration and update the schema file, and then regenerate as shown in the audited docs: https://github.com/collectiveidea/audited